### ISSUE TITLE
Save events from UI

### ISSUE THREAD

**Issue by mnu-nasir** [Serhioromano/bootstrap-calendar#772]
Is there any way to save new events from UI and save this events to the database. Please let me know.

**Comment by cablegunmaster** [comment]
Yes there is , by when saving you save the whole event to the db in the columns and rows you want it to have.
Do you have experience adapting this? and or dbs?
